src/host/outputStream.cpp

```diff
@@ -892,3 +892,17 @@ void ConhostInternalGetSet::UpdateSoftFont(const gsl::span<const uint16_t> bitPa
         pRender->UpdateSoftFont(bitPattern, cellSize, centeringHint);
     }
 }
+
+void ConhostInternalGetSet::ReparentWindow(const uint64_t handle)

```

Comment:
interesting -- why did we keep it a `uint64_t` this deep in the stack? We should decap it as early as we can